### ISSUE TITLE
Signal USR2 kills process

### ISSUE THREAD

**Issue by screentan** [go-graphite/carbonapi#426]
For both 0.12.5 and master on RHEL 7.7 sending a USR2 signal results in the process dying instead of reloading and continuing to run.  Given the contrib/carbonapi/rhel/carbonapi.service file it seems that reloading is the intent.

**Comment by Civil** [comment]
It supposed to do a graceful restart, however the library that was used for that is deprecated and archived at this moment, probably it's the reason why it's no longer working.

However it would help if you can provide more information on how it dies (e.x. if it's a clean exit, if there is anything in the logs, etc)

**Comment by screentan** [comment]
I figured it out.  When I started '/usr/local/bin/carbonapi -config /etc/carbonapi/carbonapi.yaml' and then did a 'kill -USR2' it worked correctly and the process ran with a new PID.  But doing a 'systemctl reload' failed under systemd.  The problem was the PID file path was different between carbonapi.service and carbonapi.yaml.  So having those two things set the same by default would help the next the guy.
